articles/142_idl.md

```diff
@@ -35,3 +35,45 @@ Further it describes how the interfaces are used to generate code in C, C++ and
 <div class="alert alert-warning" markdown="1">
   <b>TODO:</b> All of this too
 </div>
+
+### Type Mapping
+
+IDL types mapped to different types in different languages.
+
+
+| IDL type | C type    | C++ type    | Python type          |
+| -------- | --------- | ----------- | -------------------- |
+| bool     | \_Bool    | bool        | bool                 |
+| octet    | uint8\_t  | uint8\_t    | bytes with length 1  |
+| char     | char      | char        | str with length 1    |
+| float    | float     | float       | float                |
+| double   | double    | double      | float                |
```

Comment:
should we add long double here ?

Reply:
750fec5fd6b3c16d50c41096177421dcc9b95324